lib/api.js

```diff
@@ -164,6 +165,11 @@ PlexAPI.prototype._request = function _request(options) {
 
         resolveValue = body;
 
+        if (response.statusCode === 403 && !options.overrideToken && self.authToken) {
```

Comment:
I've tested this thoroughly manually, and never got a 403 response. Tested with different managed users with and without PIN, and different types of access to sections.

If you could provide a *detailed* description of how managed users should be setup and which URLs to request on the Plex Server, that would be really great!